MN/Algoritmi/Metode-Iterative/SOR/SOR.m

```diff
@@ -0,0 +1,31 @@
+%  Solves linear systems A*x = B using super-over-relaxation method
+%  success - 0 = solution found / 1 = maxiter exceeded
+
+function [x success] = SOR(A, b, x0, w, tol, maxiter)
+
+    %get the number of lines and columns from matrix A
+    [n n] = size(A);
+    %initialize a column vector for the solution
+    x = zeros(n, 1);
+    %I have not found the solution yet
+    succes = 1;
+  
+    while maxiter > 0
+        maxiter--;
+    
+        for i = 1:n
```

Comment:
mai jos ai 1 : i - 1.
hotaraste-te la un mod, nu le combina.
ori 1:n, 1:i-1, i+1:n peste tot
ori 1 : n, 1 : i - 1, i + 1 : n peste to